### ISSUE TITLE
Call static functions

### ISSUE THREAD

**Issue by codec-abc** [qmlnet/qmlnet#78]
Hi,

This is probably a newbie issue, but is possible to call a C# static method directly from js embedded in Javascript? If yes, could you provide a trivial example please?

**Comment by MaxMommersteeg** [comment]
Has this issue been resolved by this commit? https://github.com/qmlnet/qmlnet/commit/87df314d6d08c3b45788ac5109cac75a57a5b3c7

**Comment by pauldotknopf** [comment]
Not quite. At a C++ API level, support is there, but it isn't integrated into QML yet.
